src/main/docs/guide/security/authenticationStrategies/jwt/reader/bearerToken.adoc

```diff
@@ -1,25 +1,21 @@
-Micronaut supports https://tools.ietf.org/html/rfc6750[RFC 6750 Bearer Token] specification out-of-the-box.
-
-It is enabled by default but requires JWT Authentication to be enabled.
```

Comment:
You will need `micronaut.security.token.jwt.enabled=true`. Probably, we should not remove that sentence.

Reply:
As discussed in Slack, I think everything in this section of the docs (`15.3.3`) presumes that `micronaut.security.token.jwt.enabled=true`. So if we're repeating that setting here (`15.3.3.1`), I think for consistency we should do the same thing for `JWT Token Generation`(`15.3.3.2`) and `JWT Token Validation`(`15.3.3.3`), as they are sibling sub-sections.